front/migrations/20240927_backfill_dsv_parent_nodes.ts

```diff
@@ -0,0 +1,111 @@
+import * as _ from "lodash";
+import { Op } from "sequelize";
+
+import { getContentNodesForDataSourceView } from "@app/lib/api/data_source_view";
+import { Authenticator } from "@app/lib/auth";
+import { Workspace } from "@app/lib/models/workspace";
+import { DataSourceViewResource } from "@app/lib/resources/data_source_view_resource";
+import { DataSourceModel } from "@app/lib/resources/storage/models/data_source";
+import { DataSourceViewModel } from "@app/lib/resources/storage/models/data_source_view";
+import { makeScript } from "@app/scripts/helpers";
+
+makeScript({}, async ({ execute }, logger) => {
+  // fetch all data source views with parentsIn set to null
+  const dataSourceViews = await DataSourceViewModel.findAll({
```

Comment:
I guess we only want views that are outside of the system vault, right?

Reply:
Actually the views we want to modify are all custom views - no need to filter on  connector type and system vault ? ( we don't have any custom dsv on non managed ds or in system vault)

Reply:
That's true actually 👍🏼